plasmapy/physics/parameters.py

```diff
@@ -288,36 +299,38 @@ def ion_sound_speed(T_e,
 
     Notes
     -----
-    The ion sound speed :math:`V_S` is approximately given by
+    The ion sound speed :math:`V_S` is given by
```

Comment:
Always nice to be rid of `approximately`!

Reply:
That said, I'll make sure to add a note about approximating the function in the non-dispersive limit if the values for `k` and/or `n_e` aren't given (in response to your other question).

Reply:
Yeah, it's definitely a good idea to add that :)